static/js/components/CreateCommentForm.js

```diff
@@ -201,6 +220,49 @@ export const ReplyToCommentForm = connect(mapStateToProps, mapDispatchToProps)(
   }
 )
 
+export const EditCommentForm = connect(mapStateToProps, mapDispatchToProps)(
+  class EditCommentForm extends React.Component {
+    constructor(props) {
+      super(props)
+      this.state = {
+        patching: false
+      }
+    }
+
+    state: {
+      patching: boolean
+    }
+
+    onSubmit = async e => {
+      const { formKey, forms, patchComment, comment } = this.props
+      const { text } = R.prop(formKey, forms).value
+
+      e.preventDefault()
+
+      const updatedComment = { ...comment, text: text }
+      this.setState({ patching: true })
+      await patchComment(updatedComment)
+      this.setState({ patching: false })
+    }
+
+    render() {
+      const { forms, formKey, onUpdate, cancelReply } = this.props
+      const { patching } = this.props
```

Comment:
This should be`this.state`, the submit buttons are not currently disabling because of that.

Reply:
:woman_facepalming: